schema/crdb/dbinit.sql

```diff
@@ -827,6 +827,19 @@ CREATE UNIQUE INDEX IF NOT EXISTS lookup_ssh_key_by_silo_user ON omicron.public.
 ) WHERE
     time_deleted IS NULL;
 
+/**
+ * Represents the SSH keys copied to an instance at create time by cloud-init.
+ * Entries are added here when an instance is created (with configured SSH keys)
+ * and removed when the instance is destroyed.
+ *
+ * TODO: Should this have time created / time deleted
```

Comment:
Just commenting here since the PR is set as ready to review. This is intended to be added in this PR yes?

Reply:
Potentially, but not necessarily. Currently I have this relationship configured to be hard deleted when the instance is (soft) deleted. This was a weird case being that it's both a join _and_ not something a user can directly change. It's created at instance create and deleted at instance delete. 

If I _do_ need to add the deleted timestamp, I certainly will.